cmd/gh2gcs/cmd/root.go

```diff
@@ -225,13 +225,14 @@ func run(opts *options) error {
 	} else {
 		// TODO: Expose certain GCSCopyOptions for user configuration
 		releaseConfigs.ReleaseConfigs = append(releaseConfigs.ReleaseConfigs, gh2gcs.ReleaseConfig{
+			GCS:                *object.NewDefaultGCS(),
```

Comment:
Do we need to dereference here?

Reply:
I think it's to make a copy, which seems reasonable to me (vs having a pointer inside of the config).

Reply:
Hmm I guess it depends on how we use it. Generally I would think of creating a client, then passing an instance of it to whatever was going to use it, rather than making a copy.